signed/verifiers.go

```diff
@@ -0,0 +1,89 @@
+package signed
+
+import (
+	"crypto"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"log"
+	"reflect"
+
+	"github.com/flynn/go-tuf/Godeps/_workspace/src/github.com/agl/ed25519"
+)
+
+type Verifier interface {
+	Verify(key []byte, msg []byte, sig []byte) error
+}
+
+// Verifiers serves as a map of all verifiers available on the system and
+// can be injected into a verificationService. For testing and configuration
+// purposes, it will not be used by default.
```

Comment:
Ah no. It was at one point, then I reverted to a simpler implementation. Will fix.